File: codigo/R/stastics/graficos/tabla-frecuencias.R
## diagrama de cajas en R

# leyendo data
PSDS_PATH <- file.path(getwd())
state <- read.csv(file.path(PSDS_PATH, 'data', 'state.csv'))

breaks <- seq(from=min(state[['Population']]), 
              to=max(state[['Population']]), length=11)
pop_freq <- cut(state[['Population']], breaks=breaks, 
                right=TRUE, include.lowest=TRUE)
state['PopFreq'] <- pop_freq
table(pop_freq)

File: code/statistics/python/stat/graficos/tabla-frecuencias.py
## diagrama de cajas en R

from pathlib import Path
import pandas as pd
import matplotlib.pylab as plt

# leyendo data
DATA = Path().resolve() / 'data'

STATE_CSV = DATA / 'state.csv'
state = pd.read_csv(STATE_CSV)

# tabla de frecuencias
# El metodo `cut` separa la data

binnedPopulation = pd.cut(state['Population'], 10)
print(binnedPopulation.value_counts())
